### PR TITLE
Amend  runbook with findings from run through

### PR DESCRIPTION
**Pull request by BrianEllwood**
# Pull Request Objective

This piece of work is being tracked in
[this](https://github.com/ministryofjustice/analytical-platform/issues/4363)
GitHub Issue.

Update the documentation following the run through carried out by another team member, to clarify the process.

### CODE REVIEW

File: source/documentation/runbooks/005-analytical-platform-compute-maintenance.html.md.erb [hunk 1]
```diff
@@ -70,28 +76,21 @@ To schedule a maintenance window go to [Pagerduty Maintenance Page](https://moj-
...
 
 ##  Upgrade the EKS Nodes
 
 1. Check the `eks_node_version` from `environment-configuration.tf` against the bottlerocket [changelog](https://github.com/bottlerocket-os/bottlerocket/blob/develop/CHANGELOG.md) to see if a new version is available.
 1. If so the `eks_node_version` is formed from `'${BOTTLEROCKET_OS_RELEASE}-${FIRST_EIGHT_CHARACTERS_OF_RELEASE_SHA}'` i.e  [12.5.0](https://github.com/bottlerocket-os/bottlerocket/releases/tag/v1.25.0)-[388e1050](https://github.com/bottlerocket-os/bottlerocket/commit/388e1050a669dd2544007f2af336832b68fa0d64).
-Tip: Look at the commit URL for example `https://github.com/bottlerocket-os/bottlerocket/commit/388e1050a669dd2544007f2af336832b68fa0d64` and copy the first eight characters of the sha in this case `388e1050`.
+Tip: Go to the [Bottleroket Releases](https://github.com/bottlerocket-os/bottlerocket/releases) page and find the `latest` release to the left is the link to the  commit, follow this and look at the commit URL for example `https://github.com/bottlerocket-os/bottlerocket/commit/388e1050a669dd2544007f2af336832b68fa0d64` and copy the first eight characters of the sha in this case `388e1050`.
```
Comment (michaeljcollinsuk) [inline source/documentation/runbooks/005-analytical-platform-compute-maintenance.html.md.erb]:
```suggestion
Tip: Go to the [Bottlerocket Releases](https://github.com/bottlerocket-os/bottlerocket/releases) page and find the `latest` release to the left is the link to the commit, follow this and look at the commit URL for example `https://github.com/bottlerocket-os/bottlerocket/commit/388e1050a669dd2544007f2af336832b68fa0d64` and copy the first eight characters of the sha in this case `388e1050`.
```
Typo

File: source/documentation/runbooks/005-analytical-platform-compute-maintenance.html.md.erb [hunk 2]
```diff
@@ -101,8 +100,16 @@ Patching is a manual process. This means you will have to check each module in e
...
+1. Also check any `helm_release` for example in `helm-charts-system.tf` for any new versions.
 1. Amend the version if appropriate.
 1. Commit and push your results to the branch.
-1. Check the Terraform plan.
-1. Repeat for each module.
-1. If the complete plan is as expected request approval of the Pull Request.
+
+### Applying/Releasing the Changes
+
+Once the Terraform plan is checked and as expected, the changes can then be applied by the workflow. This needs approving  via the `Review pending deployments` of the apply job for the environment.
```
Comment (michaeljcollinsuk) [inline source/documentation/runbooks/005-analytical-platform-compute-maintenance.html.md.erb]:
```suggestion
Once the Terraform plan is checked and as expected, the changes can then be applied by the workflow. This needs approving via the `Review pending deployments` of the apply job for the environment.
```
Extra whitespace

**Review by michaeljcollinsuk — DISMISSED** [review]

File: source/documentation/runbooks/005-analytical-platform-compute-maintenance.html.md.erb [hunk 3]
```diff
@@ -15,25 +15,31 @@ This maintenance ticket includes EKS Cluster Upgrade if applicable and/or patchi
...
 ## Working on Analytical Platform Compute in Modernisation platform
 
-Be aware that due to restrictions with the state file multiple people cannot work on the environment at once. Terraform plan is fine but once you have carried out a local apply the state file cannot be used by anyone until the changes are merged.
+Be aware that due to restrictions with the state file multiple people cannot work on the environment at once. Terraform plan is fine but once you have released the apply workflow the state file cannot be used by anyone until the changes are complete. This only affects the environment you are carrying out the apply.
 
 Make the team aware and check before starting the work to avoid conflicts.
 
+### Workflows
+
+Once you create a Pull request in the Modernisation Platform reposotory, workflow will be instigated to carry the required checks etc. If you then subsequently push any changes up to the branch, you will need to go into github actions and cancel the previous workkflow so the new one can start.
```
Comment (Gary-H9) [inline source/documentation/runbooks/005-analytical-platform-compute-maintenance.html.md.erb]: typo here "reposotory", also helpful to link to the repo to be less ambiguous.

Comment (BrianEllwood) [reply source/documentation/runbooks/005-analytical-platform-compute-maintenance.html.md.erb]: @Gary-H9 fixed typo and added link to repository

File: source/documentation/runbooks/005-analytical-platform-compute-maintenance.html.md.erb [hunk 4]
```diff
@@ -15,25 +15,31 @@ This maintenance ticket includes EKS Cluster Upgrade if applicable and/or patchi
...
 
 ## The Approach
 
 1. Create a new branch / Pull Request.
 1. Make changes to the code.
-1. Create a Pull Request and check the plan.
-1. Request approval and merge the Pull Request.
+1. Create one Pull Request for all the changes.
+1. Check the Terraform plan is as expected for each environment.
+1. Request approval
```
Comment (Gary-H9) [inline source/documentation/runbooks/005-analytical-platform-compute-maintenance.html.md.erb]:
``` suggestion
1. Request approval.
```

File: source/documentation/runbooks/005-analytical-platform-compute-maintenance.html.md.erb [hunk 5]
```diff
@@ -15,25 +15,31 @@ This maintenance ticket includes EKS Cluster Upgrade if applicable and/or patchi
...
 ## Working on Analytical Platform Compute in Modernisation platform
 
-Be aware that due to restrictions with the state file multiple people cannot work on the environment at once. Terraform plan is fine but once you have carried out a local apply the state file cannot be used by anyone until the changes are merged.
+Be aware that due to restrictions with the state file multiple people cannot work on the environment at once. Terraform plan is fine but once you have released the apply workflow the state file cannot be used by anyone until the changes are complete. This only affects the environment you are carrying out the apply.
 
 Make the team aware and check before starting the work to avoid conflicts.
 
+### Workflows
+
+Once you create a Pull request in the Modernisation Platform Environments [repository](https://github.com/ministryofjustice/modernisation-platform-environments) , workflow will be instigated to carry the required checks etc. If you then subsequently push any changes up to the branch, you will need to go into github actions and cancel the previous workkflow so the new one can start.
```
Comment (Gary-H9) [inline source/documentation/runbooks/005-analytical-platform-compute-maintenance.html.md.erb]:
```suggestion
Once you create a Pull request in the Modernisation Platform Environments [repository](https://github.com/ministryofjustice/modernisation-platform-environments), workflow will be instigated to carry the required checks etc. If you then subsequently push any changes up to the branch, you will need to go into github actions and cancel the previous workkflow so the new one can start.
```

**Review by Gary-H9 — COMMENTED** [review]
Two more typos

**Review by Gary-H9 — APPROVED** [review]
Approved.

**Pull request merged**